### ISSUE TITLE
Heatmap rendering performance when many slider steps

### ISSUE THREAD

**Issue by josephernest** [plotly/plotly.js#6533]
Hi, 

This one is somehow related to https://github.com/plotly/plotly.js/issues/6521 but slightly different.

When working with 1000 slider steps, the rendering of this heatmap is quite slow (between 1 and 2 frames per second when moving the slider):  https://jsfiddle.net/wh1o2pyq/1/

When working with 50 slider steps, the rendering is super fast (more than 10 fps): https://jsfiddle.net/wh1o2pyq/2/

TL;DR: one bottleneck seems to be the sliders steps.

**Shouldn't the rendering and updating of a heatmap be independant of the slider?** Pure HTML slider object is usually super fast.

Is the slider re-rendered for each update, making this slow?

I believe that solving this issue could have massive improvement on every big Plotly/Dash dashboards using sliders :)

Any idea?

**Comment by josephernest** [comment]
(Updated with the 3d bullet point - https://jsfiddle.net/5L9ykt76/)

**Comment by morganchristiansson** [comment]
I believe I'm having the same issue with the slider but for choropleth map

I made a post in the forum about it https://community.plotly.com/t/choropleth-slider-animation-sluggish-with-many-frames/77857
